apps/journeys-admin/src/components/Editor/Toolbar/Items/StrategyItem/StrategyItem.spec.tsx

```diff
@@ -36,5 +71,13 @@ describe('StrategyItem', () => {
 
     fireEvent.click(getByRole('button'))
     await waitFor(() => expect(mockOnClick).toHaveBeenCalled())
+    expect(dispatch).toHaveBeenCalledWith({
+      type: 'SetActiveContentAction',
+      activeContent: ActiveContent.Goals
+    })
+    expect(dispatch).toHaveBeenCalledWith({
+      type: 'SetActiveSlideAction',
+      activeSlide: ActiveSlide.Content
+    })
```

Comment:
Nice work figuring out the dispatch mock! Just some minor things: 
- the `waitFor` is not needed here so we can remove that
-  Since the close menu is called after the dispatches, its good practice to have the close mock after the dispatch mock too.

```suggestion
    expect(dispatch).toHaveBeenCalledWith({
      type: 'SetActiveContentAction',
      activeContent: ActiveContent.Goals
    })
    expect(dispatch).toHaveBeenCalledWith({
      type: 'SetActiveSlideAction',
      activeSlide: ActiveSlide.Content
    })
    expect(mockOnClick).toHaveBeenCalled()
```